roles/agnosticv/templates/governor.yml.j2

```diff
@@ -66,21 +66,24 @@ spec:
 {% for __action in anarchy_governor_standard_actions
  | union((merged_vars.__meta__.deployer.entry_points | default({})).keys() | list)
 %}
+{%   if merged_vars.__meta__.deployer.entry_points[__action] is not defined or merged_vars.__meta__.deployer.entry_points[__action] | default('', true) != '' %}
```

Comment:
not quite sure to understand the logic here.


Another Q: is ` merged_vars.__meta__.deployer.entry_points[__action] is not defined` conservative enough? won't that produce an error if entry_points is not defined?

Do we need to prefix with:
```yaml
__action in  merged_vars.__meta__.deployer.entry_points|default({})
```

Reply:
No need:
```
- hosts: localhost
  connection: local
  gather_facts: false
  tasks:
  - debug:
      msg: |-
        {%   if merged_vars.__meta__.deployer.entry_points[__action] is not defined or merged_vars.__meta__.deployer.entry_points[__action] | default('', true) != '' %}
        YES
        {% else %}
        NO
        {% endif %}
    vars:
      __action: status
      merged_vars: {}
  - debug:
      msg: |-
        {%   if merged_vars.__meta__.deployer.entry_points[__action] is not defined or merged_vars.__meta__.deployer.entry_points[__action] | default('', true) != '' %}
        YES
        {% else %}
        NO
        {% endif %}
    vars:
      __action: status
      merged_vars:
        __meta__:
          deployer:
            entry_points: {}
  - debug:
      msg: |-
        {%   if merged_vars.__meta__.deployer.entry_points[__action] is not defined or merged_vars.__meta__.deployer.entry_points[__action] | default('', true) != '' %}
        YES
        {% else %}
        NO
        {% endif %}
    vars:
      __action: status
      merged_vars:
        __meta__:
          deployer:
            entry_points:
              status: null
```